resources/locale/en.yml

```diff
@@ -12,6 +12,11 @@ fof-merge-discussions:
       load_preview_button: Preview
       submit_button: => fof-merge-discussions.ref.merge
 
+    notification:
+      discussion_merged: Your discussion <b>{oldTitle}</b> was merged into this discussion by {username}.
+      preferences:
+        discussion_merged: A moderator merges one of my discussions with another
```

Comment:
A moderator implies the permission is assigned to that role. One should not make language strings too opinionated.